### ISSUE TITLE
bug: whisperplus pyannote.audio dependencies

### ISSUE THREAD

**Issue by nawed2611** [kadirnar/whisper-plus#65]
Hey, I needed some assistance in setting whisper-plus up with diarization support.

Here is the installed whisperplus specs and dependency issues

```
whisperplus==0.2.7
 - torch [required: >=2.0.0, installed: 1.13.1]
 - torchaudio [required: >=2.0.0, installed: 0.13.1]
 - pyannote.audio [required: ==3.1.0, installed: 0.0.1]
 - pyannote.core [required: ==5.0.0, installed: 4.5]
 - pyannote.database [required: ==5.0.1, installed: 4.1.3]
 - pyannote.pipeline [required: ==3.0.1, installed: 2.3]
```
 
 
 However, when running a script using whisperplus I get this -
 
```
Model was trained with pyannote.audio 0.0.1, yours is 3.1.1. Bad things might happen unless you revert pyannote.audio to 0.x.
Model was trained with torch 1.10.0+cu102, yours is 2.2.1+cu121. Bad things might happen unless you revert torch to 1.x.
```

Pls would appreciate some insight on this

**Comment by kadirnar** [comment]
I will fix this problem in version 0.0.3.

**Comment by kadirnar** [comment]
I think this is not a bug, just a warning message. You can test the model.
